src/omnicore/rules.cpp

```diff
@@ -318,8 +319,9 @@ bool ActivateFeature(uint16_t featureId, int activationBlock, int transactionBlo
     const CConsensusParams& params = ConsensusParams();
 
     // check activation block is allowed
-    if ((activationBlock < (transactionBlock + params.MIN_ACTIVATION_BLOCKS)) ||
-        (activationBlock > (transactionBlock + params.MAX_ACTIVATION_BLOCKS))) {
+    if ( ((transactionBlock + params.MIN_ACTIVATION_BLOCKS) > activationBlock) ||
```

Comment:
Ahh, sorry leftover from my testing/debugging I think as feature activations weren't working on testnet...  

FYI on testnet when you changed ```MAX_ACTIVATION_BLOCKS``` to ```std::numeric_limits<int>::max()``` this means that ```transactionBlock + params.MAX_ACTIVATION_BLOCKS``` overflows and then all activations fail the notice check. I changed it to 999999 which sorted it out, just need to clean up after myself hehe :)